tests/features/common/kie-workbench-common.feature

```diff
@@ -116,4 +116,48 @@ Feature: Decision/Business Central common features
 
   Scenario: RHPAM-3517: Update maven to 3.6
     When container is started with command bash
-    Then run sh -c "mvn --version | sed -n -e 's/^.*Apache //p' | grep 3.6 && echo  all good" in container and check its output for all good
\ No newline at end of file
+    Then run sh -c "mvn --version | sed -n -e 's/^.*Apache //p' | grep 3.6 && echo  all good" in container and check its output for all good
+
+  Scenario: Test if KIE Server access is correct set with user/pass
+    When container is started with env
+      | variable       | value     |
+      | KIE_ADMIN_USER | superUser |
+      | KIE_ADMIN_PWD  | @w3s0m3   |
+    Then container log should contain -Dorg.kie.server.user=superUser
+     And container log should contain -Dorg.kie.server.pwd=@w3s0m3
+     And container log should not contain -Dorg.kie.server.token
+
+  Scenario: Test if KIE Server access is correct set with token
+    When container is started with env
+      | variable         | value             |
+      | KIE_ADMIN_USER   | superUser         |
+      | KIE_ADMIN_PWD    | @w3s0m3           |
+      | KIE_SERVER_TOKEN | some-random-token |
+    Then container log should not contain -Dorg.kie.server.user=superUser
+     And container log should not contain -Dorg.kie.server.pwd=@w3s0m3
```

Comment:
In this scenario log should not contain any Kie Server user/pass. Just to be sure it would be better to check that these system properties are not used in log (without values) - sane is check token property in previous test. 
```suggestion
    Then container log should not contain -Dorg.kie.server.user
     And container log should not contain -Dorg.kie.server.pwd
```